src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  switch (true) {
+    case days < 3:
+      return days * 40;
+    case days >= 3 && days < 7:
+      return days * 40 - 20;
+    case days >= 7:
+      return days * 40 - 50;
```

Comment:
The use of a switch statement with 'true' as the expression is unconventional and can be confusing. It would be better to use if-else statements here to improve readability and follow the checklist.